pubsub/subscriber_rabbitmq.go

```diff
@@ -137,6 +138,15 @@ func (r *rabbitMQWorker) initConn(queue, exchange string) error {
 		log.Panicln("worker could not declare rabbitmq queue", queue, err)
 	}
 
+	err = ch.Qos(
+		3,     // prefetch count
```

Comment:
could you set this to 1 first? 
I might expose this as an option for max inflight later